api/src/main/java/org/jfrog/artifactory/client/ArtifactoryStreamingResponse.java

```diff
@@ -0,0 +1,15 @@
+package org.jfrog.artifactory.client;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+
+/**
+ * ArtifactoryStreamingResponse object returned from {@link Artifactory#streamingRestCall(ArtifactoryRequest)}.
+ * acts as a wrapper for {@link org.apache.http.HttpResponse}.
+ */
+public interface ArtifactoryStreamingResponse extends BaseArtifactoryResponse {
```

Comment:
Suggestion - please consider making ArtifactoryStreamingResponse AutoClosable to allow using easily it with try-with-resources

Reply:
done